README.md

```diff
@@ -14,6 +14,29 @@ Parameters in some Income Tax (Making Tax Digital) APIs map to box numbers in [S
 
 ---
 
+### 4 Jun 2024
+
+The following changes are now available in Sandbox and Production.
+
+#### Property Business API
+
+* Update the `consolidatedExpenses` parameter description in all versions of the API documentation for endpoints:
```

Comment:
Can we say 'field' rather than 'parameter' (also for self-employment) since this is a payload data item (also it is part of the output on a retrieval, but 'parameter' suggests input).